configlib/config.go

```diff
@@ -206,11 +206,12 @@ func remove(ymlfile string, packageName string) error {
 }
 
 // SetCustomizations ... set ENV values in Rsettings
-func SetCustomizations(cfg PkgrConfig, rSettings *rcmd.RSettings) {
+func SetCustomizations(cfg PkgrConfig, rSettings rcmd.RSettings) rcmd.RSettings {
```

Comment:
generally, when returning a modified an object we'd want the object itself to be the first arg

SetCustomizations(rSettings, cfg)...

Reply:
done.

I usually put "in"s first and then the "out"s; in this case rSettings is kinda an "out". (Some languages have an out keyword to make it clear)